integration-cli/docker_cli_ps_test.go

```diff
@@ -256,10 +160,8 @@ func assertContainerList(out string, expected []string) bool {
 
 func (s *DockerSuite) TestPsListContainersSize(c *check.C) {
 
-	cmd := exec.Command(dockerBinary, "run", "-d", "busybox", "echo", "hello")
-	runCommandWithOutput(cmd)
-	cmd = exec.Command(dockerBinary, "ps", "-s", "-n=1")
-	baseOut, _, err := runCommandWithOutput(cmd)
+	exec.Command(dockerBinary, "run", "-d", "busybox", "echo", "hello").Run()
```

Comment:
I thought to do so, but I preferred to not change it's behaviour. I'm using dockerCmd now.